test/reconnect.ts

```diff
@@ -138,8 +138,8 @@ describe('reconnect', function() {
         var extra_session_opens: number = 0;
         var disconnects: number = 0;
         var c: rhea.Connection = container.connect(listener.address());
-        var s: rhea.Sender = c.open_sender('my-sender');
         var extra_session: rhea.Session = c.create_session();
+        var s: rhea.Sender = c.open_sender('my-sender');
```

Comment:
I commented out the `extra_session.remove();` line in this test case and was expecting the test to fail. But it didn't.

Since the sender and therefore its session gets created first, the `sender_open` event gets fired and calls `done()` before the connection attempts to reconnect the `extra_session`

By swapping the order, the test now works as expected. Commenting out the line `extra_session.remove();` now fails the test as expected